app/views/customers/_form.html.erb

```diff
@@ -0,0 +1,63 @@
+<%= form_for @customer do |f| %>
+  <%= f.label :full_name %>
+  <%= f.text_field :full_name %>
+
+  <%= f.label :cpf %>
+  <%= f.text_field :cpf %>
+
+  <%= f.label :birth_date %>
+  <%= f.text_field :birth_date %>
+
+  <%= f.label :rg %>
+  <%= f.text_field :rg %>
+
+  <%= f.label :rg_sender %>
+  <%= f.text_field :rg_sender %>
+
+  <%= f.label :rg_expedition %>
+  <%= f.text_field :rg_expedition %>
+
+  <%= f.label :place_birth %>
+  <%= f.text_field :place_birth %>
+
+  <%= f.label :place_birth_uf %>
+  <%= f.text_field :place_birth_uf %>
+
+  <%= f.label :nacionality %>
+  <%= f.text_field :nacionality %>
+
+  <%= f.label :mother %>
+  <%= f.text_field :mother %>
+
+  <%= f.label :father %>
+  <%= f.text_field :father %>
+
+  <%= f.label :postal_code %>
+  <%= f.text_field :postal_code %>
+
+  <%= f.label :address %>
+  <%= f.text_field :address %>
+
+  <%= f.label :neighborhood %>
+  <%= f.text_field :neighborhood %>
+
+  <%= f.label :adjunct %>
+  <%= f.text_field :adjunct %>
+
+  <%= f.label :city %>
+  <%= f.text_field :city %>
+
+  <%= f.label :state %>
+  <%= f.text_field :state %>
+
+  <%= f.label :phone %>
+  <%= f.text_field :phone %>
+
+  <%= f.label :cell_phone %>
+  <%= f.text_field :cell_phone %>
+
+  <%= f.label :email %>
+  <%= f.text_field :email %>
+
+  <%= f.submit %>
+<% end %>
```

Comment:
For a app that it has too many forms or forms can be too big I enjoy simple_form gem.